test/mockupdb/test_reset_and_request_check.py

```diff
@@ -45,7 +45,7 @@ def responder(request):
         kwargs = {'socketTimeoutMS': 100}
         # Disable retryable reads when pymongo supports it.
         kwargs['retryReads'] = False
-        self.client = MongoClient(self.server.uri, **kwargs)
+        self.client = MongoClient(self.server.uri, **kwargs)  # type: ignore
```

Comment:
What's up with the type ignore here? It seems perfectly reasonable to pass `**kwargs` to MongoClient because it accepts `**kwargs`. Is this something we should report to mypy?

Reply:
Yeah it is a type limitation of mypy, I'll look for/file an issue

Reply:
Found this [comment](https://github.com/python/mypy/issues/11084#issuecomment-917426934): I'll try `TypedDict`

Reply:
Actually, it really is a bug, already reported: https://github.com/python/mypy/issues/8862

Reply:
Created https://jira.mongodb.org/browse/PYTHON-3105 to track this